### ISSUE TITLE
Properly namespace all remaining used translation keys

### ISSUE THREAD

**Issue by akatov** [status-im/status-react#17811]
# Problem

#17610 introduced linting of the usage of translation keys in `i18n/label` expressions through clj-kondo. This linting rule requires translation keywords to use the `:t/` namespace.

However, this method does not catch all the usages of `i18n/label` (e.g. through intermediary functions). #17735 created a script to (amongst other things) find such occurrences, but their actual fixing has been moved to this separate issue as there are so many of them (there are still 388 non-namespaced keywords in `src` that match the keys in `translations/en.json`.)

We would like all of the ones actually used for translation to be namespaced with `t:/`.

## List of  keys
Here are the keys in `translations/en.json` that are probably used as non-namespaced keywords in `src`:

```
about-key-storage-content
about-key-storage-title
about-names-content
about-names-title
access-key
account-added
accounts
active-online
active-unknown
add-an-account
add-contact
add-nickname
added
advanced-settings
agree-by-continuing
ask-in-status
available
available-participants
balance
biometric-auth-reason-verify
biu
block-contact
bold
browsed-websites
browsers
cancel-keycard-setup
cannot-read-card
card-is-blank
card-reseted
category
change-logging-enabled
change-passcode
change-password
changed-amount-warning
changed-asset-warning
chaos-mode
chaos-unicorn-day
chaos-unicorn-day-details
chat-name
chat-settings
chats
clear-history-confirmation
close-app-content
command-button-send
community-color
community-info
complete-hardwallet-setup
completed
confirm-new-password
confirmations-helper-text
connecting-requires-login
connection-with-the-card-lost
connection-with-the-card-lost-setup-text
connection-with-the-card-lost-text
contact-s
copy-info
copy-to-clipboard
create-community
create-multiaccount
create-pin
create-pin-description
created-group-chat-description
cryptokitty-name
currency
current
current-network
current-password
cut
datetime-day
datetime-hour
datetime-minute
datetime-second
decryption-failed-content
default
delete-and-leave-group
delete-message
delete-my-account
description
dev-mode
dev-mode-settings
disable
disabled
disconnected
edit-group
edit-nickname
edit-profile
empty-chat-description
empty-chat-description-one-to-one
empty-chat-description-public
empty-chat-description-public-share-this
encrypt-with-password
ens-primary-username
ens-remove-hints
ens-show-username
ens-test-message
enter-12-words
enter-pin
enter-seed-phrase
error-unable-to-get-balance
error-unable-to-get-prices
error-unable-to-get-token-balance
errors
eth
ethereum-node-started-incorrectly-description
ethereum-node-started-incorrectly-title
export-account
export-key
favourite
find
finishing-card-setup
fr
generate-a-key
generate-a-new-key
generate-account
generating-codes-for-pairing
generating-mnemonic
get-status-at
go-to-settings
got-it
group-chat
group-chat-admin-added
group-chat-created
group-chat-member-added
group-chat-member-joined
group-chat-member-removed
group-chat-name-changed
help
help-center
hold-card
home
hooks
identifier
image-remove-current
image-source-gallery
image-source-make-photo
image-source-title
import-community
in-contacts
include
incorrect-code
initialization
intro-message1
intro-privacy-policy-note1
intro-privacy-policy-note2
intro-text
intro-text1
intro-text2
intro-text3
intro-title1
intro-title2
intro-title3
intro-wizard-text1
intro-wizard-text2
intro-wizard-text3
intro-wizard-text6
intro-wizard-title-alt4
intro-wizard-title-alt5
intro-wizard-title1
intro-wizard-title2
intro-wizard-title3
invalid-key-confirm
invalid-key-content
invalid-range
invited
italic
join-group-chat
join-group-chat-description
joined-group-chat-description
keycard-desc
keycard-onboarding-finishing-header
keycard-onboarding-pairing-header
keycard-onboarding-preparing-header
keycard-onboarding-start-header
keycard-onboarding-start-step1
keycard-onboarding-start-step1-text
keycard-onboarding-start-step2
keycard-onboarding-start-step2-text
keycard-onboarding-start-step3
keycard-onboarding-start-step3-text
keycard-onboarding-start-text
language
left
les-ulc
loading
logging
logging-enabled
mailserver-connection-error
mailserver-reconnect
mailserver-request-error-status
max-fee
members
members-active
members-active-none
message-not-sent
message-options-cancel
might-break
migrations-failed-content
mobile-network-continue-syncing
mobile-network-sheet-configure
mobile-network-sheet-remember-choice
mobile-network-sheet-settings
mobile-network-stop-syncing
mobile-network-stop-syncing-details
mobile-syncing-sheet-details
mobile-syncing-sheet-title
multiaccounts-recover-enter-phrase-text
mute-chat
mute-group
my-profile
network-invalid-network-id
network-invalid-status-code
network-invalid-url
new-contact
new-group
new-password
new-public-group-chat
no-pairing-slots-available
no-result
no-tokens-found
node-info
not-found
notify
off
open-dapp
optional
pairing
pairing-card
passphrase
password-placeholder2
password_error1
paste-json
pay-to-chat
pending-confirmation
phone-e164
photos
pin
pin-code
pinned-by
preview-privacy
product-information
public-chat
public-chats
public-group-status
public-group-topic
public-key
puk-and-pairing-codes-displayed
quiet-days
quiet-hours
re-encrypt-key
receive-transaction
recipient-code
recover
recover-keycard-multiaccount-not-supported
recover-with-keycard
recovering-key
recovery-confirm-phrase
recovery-success-text
recovery-typo-dialog-description
recovery-typo-dialog-title
remember-me
remove-network
removed
request-access
required-field
resend-message
save-password
save-password-unavailable
save-password-unavailable-android
see-details
select
select-chat
selected
send-message
send-request
send-request-amount-max-decimals
send-request-unknown-token
send-sending-to
sent-at
share-chat
share-contact-code
share-image
share-my-profile
share-profile-link
share-public-chat-text
sharing-copy-to-clipboard
sharing-share
show-less
show-more
sign-message
sign-you-in
signing
simple
soon
start-chat
start-conversation
start-group-chat
start-new-chat
status
status-hardwallet
status-not-sent-click
status-not-sent-tap
status-pending
status-sent
status-tx-not-found
strikethrough
submit
success
sync-code-generated
sync-in-progress
sync-synced
tag-was-lost
text-input-disabled
this-device-desc
this-will-take-few-seconds
to-block
to-encrypt-enter-password
to-see-this-message
token-auto-validate-decimals-error
token-auto-validate-name-error
token-auto-validate-symbol-error
topic-name-error
transaction
transaction-history
transaction-request
transactions-filter-select-all
transactions-filter-title
transactions-history
transactions-history-empty
transactions-history-loading
tribute-required-by-multiaccount
tribute-state-paid
tribute-state-pending
tribute-state-required
tribute-to-talk
tribute-to-talk-add-friends
tribute-to-talk-are-you-friends
tribute-to-talk-ask-to-be-added
tribute-to-talk-contact-received-your-tribute
tribute-to-talk-desc
tribute-to-talk-disabled
tribute-to-talk-disabled-note
tribute-to-talk-enabled
tribute-to-talk-finish-desc
tribute-to-talk-learn-more-1
tribute-to-talk-learn-more-2
tribute-to-talk-learn-more-3
tribute-to-talk-paywall-learn-more-1
tribute-to-talk-paywall-learn-more-2
tribute-to-talk-paywall-learn-more-3
tribute-to-talk-pending
tribute-to-talk-pending-note
tribute-to-talk-removing-note
tribute-to-talk-set-snt-amount
tribute-to-talk-signing
tribute-to-talk-transaction-failed-note
tribute-to-talk-tribute-received1
tribute-to-talk-tribute-received2
tribute-to-talk-you-require-snt
ulc-enabled
unblock-contact
unknown-status-go-error
unmute-chat
unmute-group
unpin
url
usd-currency
use-valid-contact-code
view-cryptokitties
view-cryptostrikers
view-etheremon
view-gitcoin
view-superrare
waiting-for-wifi
waiting-for-wifi-change
waiting-to-sign
wallet-connect
wallet-settings
warning-message
web-view-error
welcome-screen-text
welcome-to-status
welcome-to-status-description
word-count
write-down-and-store-securely
wrong-contract
you-can-change-account
your-data-belongs-to-you
your-data-belongs-to-you-description
```

**Note**: see [this comment](https://github.com/status-im/status-mobile/pull/17735#issuecomment-1793437127) - after #17814 there should only be 41 keys left.

## Implementation

Uncomment the relevant expressions in `scripts/lint_translations.clj`, in particular the line to print out the individual occurrences of the non-namespaced keyword usages.

Inspect the code to find where 

## Notes

Even once `scripts/lint_translations.clj` does not print "Probably non-namespaced translation keyword usage" warnings any more, it does not mean that there still aren't any non-namespaced keywords passed to `i18n/label`. It simply means, that any translation key in `translations/en.json` that's actually used inside `src`, is used with the namespace `:t/` at least once.

We would still need a `:pre` condition for `i18n/label`, or malli instrumentation to catch all the bad usages. See https://github.com/status-im/status-mobile/pull/17735#issuecomment-1792593032

## Future Steps

Once all non-namespaced translation keyword usages have been resolved, we can proceed to removing all unused translation keys (#17813).

Once both #17811 and #17813 are solved, we can remove the functionality to list non-namespaced translation keys from `scripts/lint_translations.clj`.

**Comment by J-Son89** [comment]
@ilmotta - I can't recall - is this issue already handled?

**Comment by ilmotta** [comment]
> @ilmotta - I can't recall - is this issue already handled?

I believe so @J-Son89. But we may prefer to close it now, unless we want to keep it open for external contribution.
